packages/vscode-extension/src/officeDevHandlers.ts

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author xurui yao <xuruiyao@microsoft.com>
+ */
+"use strict";
+
+import * as vscode from "vscode";
+import * as fs from "fs-extra";
+import {
+  OfficeDevTerminal,
+  triggerGenerateGUID,
+  triggerInstall,
+  triggerStopDebug,
+  triggerValidate,
+} from "./debug/taskTerminal/officeDevTerminal";
+import {
+  FileNotFoundError,
+  fetchManifestList,
+  globalStateGet,
+  globalStateUpdate,
+} from "@microsoft/teamsfx-core";
+import { VS_CODE_UI } from "./extension";
+import { FxError, Result, Warning, err, ok } from "@microsoft/teamsfx-api";
+import * as globalVariables from "./globalVariables";
+import * as path from "path";
+import { localize } from "./utils/localizeUtils";
+import {
+  ShowScaffoldingWarningSummary,
+  autoInstallDependencyHandler,
+  openReadMeHandler,
+  openSampleReadmeHandler,
+  showLocalDebugMessage,
+} from "./handlers";
+import { Uri } from "vscode";
+import { isTriggerFromWalkThrough } from "./utils/commonUtils";
+import { GlobalKey } from "./constants";
+import { TelemetryTriggerFrom } from "./telemetry/extTelemetryEvents";
+
+export async function openOfficePartnerCenterHandler(
+  args?: any[]
+): Promise<Result<boolean, FxError>> {
+  const url = "https://aka.ms/WXPAddinPublish";
+  return VS_CODE_UI.openUrl(url);
+}
+
+export async function openGetStartedLinkHandler(args?: any[]): Promise<Result<boolean, FxError>> {
+  const url = "https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins";
```

Comment:
Please use AKA link to prevent invalid URL in future.

Reply:
We'll fix the hard coded link to soft link in the future as the following work items. Thanks for your comments. It's very important.